packages/web/package.json

```diff
@@ -37,6 +37,11 @@
       "types": "./dist/components/ServerInject.d.ts",
       "default": "./dist/components/ServerInject.js"
     },
+    "./request": {
+      "require": "./dist/cjs/server/request.js",
```

Comment:
We need this so that typescript doesn't complain in projects, but in reality the CJS version should never be used.